js/statistics.js

```diff
@@ -20,13 +20,13 @@ function loadStatistics(year, stats) {
     statistics.class.textContent = year;
     statistics.stats.students.textContent = students.get(year).length;
 
-    if (!stats.get(year)['Number of destinations']) {
+    if (!stats.get(parseInt(year))['Number of destinations']) {
```

Comment:
Wow does papaparse properly parse integers?

Reply:
Since that's kinda cool, maybe we want to just store `year` as an integer in the first place.

Reply:
Strangely, stats.get(year) was undefined. This part only stopped giving me errors when I used integers. Not sure why